### PR TITLE
chore: documentation of new filter in db-object widget

### PR DESCRIPTION
**Pull request by latin-panda**
# Description

Documentation of new filter in db-object widget
[CHT Core PR](https://github.com/medic/cht-core/pull/8759)

# License

The software is provided under AGPL-3.0. Contributions to this project are accepted under the same license.

### CODE REVIEW

**Comment by latin-panda** [comment]
@michaelkohn I need help explaining this 😅 please have a look

**Comment by latin-panda** [comment]
Hi Binod, I've added new documentation as part of [this work](https://github.com/medic/cht-core/pull/8759). Can you please have a look and let me know if it's clear? Thanks!

File: content/en/apps/guides/forms/form-inputs.md [hunk 1]
```diff
@@ -145,6 +145,14 @@ A value can be pre-selected for the search box via a `calculate` expression. By
 | string      | person_id    | Person ID    | select-contact type-person |                              |
 | string      | household_id | Household ID | select-contact type-clinic | ../inputs/contact/parent/_id |
 
+### Loading descendants of the current contact
+
+Use the appearance `descendant-of-current-contact` to load the current contact's descendant when opening an app form or contact form from the Contact's tab.
```
Comment (binokaryg) [inline content/en/apps/guides/forms/form-inputs.md:150]:
```suggestion
Use the appearance `descendant-of-current-contact` to load the current contact's descendants when opening an app form or contact form from the "People" tab.
```
This is to make it consistent with other references to the "People" tab throughout the page.

**Review by binokaryg — APPROVED** [review]
Thanks for adding this section. I've suggested an optional change for consistency.

**Pull request merged**